cmd/soroban-cli/src/commands/keys/default.rs

```diff
@@ -4,13 +4,23 @@ use crate::{commands::global, config::locator, print::Print};
 pub enum Error {
     #[error(transparent)]
     Config(#[from] locator::Error),
+
+    #[error("Identify name is required unless --clear is specified")]
+    NameRequired,
+
+    #[error("Identify name cannot be used with --clear")]
+    NameWithClear,
 }
 
 #[derive(Debug, clap::Parser, Clone)]
 #[group(skip)]
 pub struct Cmd {
     /// Set the default network name.
-    pub name: String,
+    pub name: Option<String>,
+
+    /// Clear the default source account.
+    #[arg(long)]
+    pub clear: bool,
```

Comment:
Should we omit the clear: bool, and just make it if name is None, then clear it?